lib/auth/webauthncli/api.go

```diff
@@ -180,26 +180,9 @@ func crossPlatformLogin(
 	ctx context.Context,
 	origin string, assertion *wantypes.CredentialAssertion, prompt LoginPrompt, opts *LoginOpts,
 ) (*proto.MFAAuthenticateResponse, string, error) {
-	if isLibfido2Enabled() {
```

Comment:
Could you delete the isLibfido2Enabled function too?

Note that it appears on both fido2.go and fido2_other.go.